### PR TITLE
84 multiinstance props

### LINKED ISSUE #84: Migrate BPMN generic multiInstance group

### PR DESCRIPTION
**Pull request by MaxTru**
closes #84

Screenshot_20210827_195506

Screenshot_20210830_095845

Note that this PR only includes the bpmn generic multiInstanceProps. Next step: I will follow-up with #85

### CODE REVIEW

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp; LGTM

Note: I'll reach out to TechWriting since we write multi-instance [with](https://docs.camunda.io/docs/0.25/product-manuals/zeebe/bpmn-workflows/multi-instance/multi-instance) and [without](https://camunda.com/best-practices/modeling-with-situation-patterns/#using-a-multi-instance-task) a hyphen in our docs.

**Comment by andreasgeier** [comment]
Checked with TechWriter: we should use a hyphen. You could fix the sentence case now `Multi-instance` or we do it later in the finishing.

**Review by pinussilvestrus — COMMENTED** [review]
Great stuff 👍  Only one minor thing, then we can merge it.

File: src/provider/bpmn/properties/MultiInstanceProps.js [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+/**
+ * getProperty - get a property value of the loop characteristics.
+ *
+ * @param {djs.model.Base} element
+ * @param {string} propertyName
+ *
+ * @return {any} the property value
+ */
+function getProperty(element, propertyName) {
+  var loopCharacteristics = getLoopCharacteristics(element);
```
[the remark this answers is not in the document]
Comment (MaxTru) [reply src/provider/bpmn/properties/MultiInstanceProps.js]: Upsi - that is a clear leftover from the old panel :-) Corrected 

Thanks for catching!

**Review by pinussilvestrus — APPROVED** [review]

**Pull request merged**